go/go.mod

```diff
@@ -19,7 +19,7 @@ require (
 	github.com/morikuni/aec v1.0.0 // indirect
 	github.com/prometheus/client_golang v1.7.1
 	github.com/prometheus/common v0.10.0
-	github.com/prometheus/prometheus v2.5.0+incompatible
+	github.com/prometheus/prometheus v1.8.2-0.20190525122359-d20e84d0fb64
```

Comment:
Corresponds to prometheus [version 2.10.0](https://github.com/prometheus/prometheus/tree/v2.10.0)